### PR TITLE
Added uglified definition of node.js module to dist/cartodb.js

### PR DESCRIPTION
**Pull request by alonsogarciapablo**
Part of #495.

@viddo what do you think of this? Thanks!

### CODE REVIEW

**Comment by viddo** [comment]
Hmm not sure it works as intended. I tried installing this branch locally to try it out and get:

```
 node test.js

/Users/viddo/src/tmp/node_modules/cartodb.js/dist/cartodb.js:5
e.amd&&define.amd.jQuery&&define("jquery",[],function(){return J})}(window),fu
                                                                    ^
ReferenceError: window is not defined
    at /Users/viddo/src/tmp/node_modules/cartodb.js/dist/cartodb.js:5:30256
    at Object.<anonymous> (/Users/viddo/src/tmp/node_modules/cartodb.js/dist/cartodb.js:21:11299)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/viddo/src/tmp/test.js:1:73)
    at Module._compile (module.js:456:26)
```

where `test.js` contains:
```
var cdb = require('cartodb.js');
console.log(cdb)
```

**Comment by viddo** [comment]
Apart from that looks legit :+1:

File: src/node.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+if (typeof module !== 'undefined' && typeof module.exports !== 'undefined') {
```
Comment (alonsogarciapablo) [inline src/node.js]: I guess this doesn't make sense if we don't want to actually export a module.

**Comment by alonsogarciapablo** [comment]
I'm closing this after talking to @javisantana about it. We might make cartodb.js work as an npm module in future versions. Thanks for your help @viddo! I learnt a bit more about npm! :)

**Pull request closed without merging**